src/nodejs_bindings.cpp

```diff
@@ -245,8 +265,20 @@ NAN_METHOD(Annotator::annotateRouteFromLonLats)
 
         void Execute() override
         {
-            const auto internalIds = self.annotator->coordinates_to_internal(coordinates);
-            wayIds = self.annotator->annotateRoute(internalIds);
+            try
+            {
+                const auto internalIds = self.annotator->coordinates_to_internal(coordinates);
+                wayIds = self.annotator->annotateRoute(internalIds);
+            }
+            catch (const RouteAnnotator::RtreeError &e)
+            {
+                std::cerr << e.what() << std::endl;
```

Comment:
The printing to cerr here needs to be removed; in the Node.js bindings you should not print to stdout; you're running inside v8, so using the Javascript-bound error features is all you are allowed to do here.

Reply:
:fearful: